api/report_server.thrift

```diff
@@ -443,7 +450,7 @@ service codeCheckerDBAccess {
   // for all of the runs and in compare mode all of the runs
   // will be used as a baseline excluding the runs in compare data.
   // PERMISSION: PRODUCT_ACCESS
-  map<string, i64> getCheckerCounts(1: list<i64> runIds,
+  CheckerDataCounts getCheckerCounts(1: list<i64> runIds,
                                     2: ReportFilter_v2 reportFilter,
```

Comment:
The length of the type changed, please align the arguments to be under each other (introduce a single space).